src/ContaoManager/Plugin.php

```diff
@@ -84,6 +86,17 @@ public function registerContainerConfiguration(LoaderInterface $loader, array $m
     }
 
     /**
+     * {@inheritdoc)
+     */
+    public function getFirewallConfig(array $config)
+    {
+        return array_merge(
+            $config,
+            Yaml::parse(file_get_contents(__DIR__.'/../Resources/contao-manager/security_firewalls.yml'))
```

Comment:
It has to be an array so you can order it. That's the whole point about this plugin :)